charts/sn-platform/values.yaml

```diff
@@ -1487,6 +1487,38 @@ datadog:
 ##
 grafana:
   component: grafana
+  grafana.ini:
+    paths:
+      data: /var/lib/grafana/pulsar/data
+      plugins: /var/lib/grafana/pulsar/plugins
+      provisioning: /var/lib/grafana/pulsar_provisioning
+    server:
+      domain: "{{ GRAFANA_DOMAIN }}"
+      serve_from_sub_path: "{{ GRAFANA_SERVE_FROM_SUB_PATH }}"
+      root_url: "{{ GRAFANA_ROOT_URL }}"
+    analytics:
+      check_for_updates: true
+    security:
+      admin_user: "{{ GRAFANA_ADMIN_USER }}"
+      admin_password: "{{ GRAFANA_ADMIN_PASSWORD }}"
+    auth.azuread:
+      name: Azure AD
+      enabled: false
+      allow_sign_up: true
+      client_id: ""
+      client_secret: ""
+      scopes: openid email profile
+      auth_url: ""
+      token_url: ""
+      allowed_domains: ""
+      allowed_groups: ""
```

Comment:
Make it possible to set `role_attribute_strict = true` to map users to Grafana roles based on Azure AD App Roles.